examples/benchmark_lazy_eager_loading.py

```diff
@@ -216,7 +221,7 @@ def run_training(model, dataloader, loss, optimizer, n_epochs=1, cuda=False):
 # Next, we define the different hyperparameters that we want to compare:
 
 PRELOAD = [True, False]  # True -> eager loading; False -> lazy loading
-N_SUBJECTS = [10]  # Number of recordings to load from the TUH Abnormal corpus
+N_RECORDINGS = [10]  # Recording ids to load from the TUH Abnormal corpus
```

Comment:
Is this now recording ids or number of recordings?

Reply:
Thank you, @robintibor. You are correct. I updated the comment.

Reply:
Number of recordings